examples/restclient/src/main/kotlin/cnt/adapters/restclient/model/Actor.kt

```diff
@@ -0,0 +1,14 @@
+package cnt.adapters.restclient.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.quarkus.runtime.annotations.RegisterForReflection
+
+@RegisterForReflection
+data class Actor(
+    @field:JsonProperty("id")
```

Comment:
Just curious: Is the `field:JsonProperty `really needed if the field has actually the same name?

Reply:
no